dds/DCPS/TypeObject.cpp

```diff
@@ -4,9 +4,12 @@
  */
 
 #include "DCPS/DdsDcps_pch.h" //Only the _pch include should start with DCPS/
+#include "dds/DCPS/Message_Block_Ptr.h"
 
 #include "TypeObject.h"
 
+#include "dds/DdsDcpsCoreC.h"
+
 #include "Hash.h"
```

Comment:
```suggestion
#include "TypeObject.h"

#include "Message_Block_Ptr.h"
#include "Hash.h"

#include <dds/DdsDcpsCoreC.h>
```

According to https://github.com/objectcomputing/OpenDDS/blob/master/docs/guidelines.md#includes